middleware.js

```diff
@@ -265,18 +269,18 @@ module.exports = function(options) {
 
     // Compare mtimes
     fs.stat(sassPath, function(err, sassStats) {
-      if (err) {
-        error(err);
```

Comment:
>  that seems redundant now

Actually, this check is bit redundant because `compile()` has other callsites where we are not ensuring whether the file exists. We should error out (or log with debug severity) in compile function instead.

But lets keep it as is for now; I am planning to but some debug logs here and other places with new tag `skip`, to indicate that middleware recognized the request but since it can't fetch file stat, skipping to next. The issue https://github.com/sass/node-sass-middleware/issues/88 will ultimately get fixed once we also log a message and skip to next middleware when middleware receives a mismatching prefixed request (in case user has used the prefix option); so in the OP's example `outdatedbrowser.min.css` will not match the prefix, hence we skip to next middleware but still debug log that we (the middleware) saw the request and skipping to next due to such and such reasons.

This will help consumer debugging.